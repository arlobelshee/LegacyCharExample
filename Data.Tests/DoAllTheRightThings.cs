using ApprovalTests;
using Data.PipelineSynchronous;
using NUnit.Framework;

namespace Data.Tests
{
	[TestFixture]
	public class DoAllTheRightThings
	{
		[Test]
		public void OrchestrationShouldBeAsExpected()
		{
			Collector<CharacterFile> characterTrap;
			Collector<ConfigFile> configTrap;
			PipeMiddle<CharacterFile, ConfigFile> configFileNode;
			Collector<CardData> partialCardsTrap;
			Collector<CardData> localCardsTrap;
			var orchestration = DoEverything.CreateMorePipe(out characterTrap, out configTrap, out configFileNode, out partialCardsTrap, out localCardsTrap);
			Approvals.Verify(orchestration);
		}
	}
}
