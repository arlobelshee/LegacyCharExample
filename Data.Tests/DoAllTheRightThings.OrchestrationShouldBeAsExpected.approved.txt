--> CharacterFile CharacterFile.From(String fileName)
 |--> Collector
 |--> ConfigFile ConfigFile.Matching(CharacterFile characterFile)
 | |--> Collector
 | |--> Void <>c__DisplayClass0_0`2.<Scatter>b__0(ConfigFile source, Action`1 handleData)
 | | |--> Collector
 |--> Void <>c__DisplayClass0_0`2.<Scatter>b__0(CharacterFile source, Action`1 handleData)
 | |--> Collector
